controller/array_action/svc_cli_result_reader.py

```diff
@@ -0,0 +1,117 @@
+from controller.common.csi_logger import get_stdout_logger
+import controller.array_action.errors as controller_errors
+
+logger = get_stdout_logger()
+
+ID_PARAM_NAME = "id"
+
+
+class SVCListResultsReader:
+    """
+    Iterable object used to read raw command response from m SVS array.
```

Comment:
typo - "from m and SVS" should be "from SVC"

Reply:
fixed

Reply:
if it's fixed, one of you should resolve this